### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by piotr-yuxuan** [cemerick/url#30]
Hi @cemerick,

Thanks for this project, it's been very useful! Just asking, do you still have time to maintain it?

**Comment by piotr-yuxuan** [comment]
As mentionned [here](https://github.com/cemerick/url/issues/28#issuecomment-380590249), better to look to a fork of it: https://github.com/arohner/uri

**Comment by cemerick** [comment]
No, not really. Yes, please enjoy the forks. I'm archiving this repo which should clarify things, but will leave this issue open so its question and my answer are clearly seen.
